Composer/packages/client/src/recoilModel/dispatchers/publisher.ts

```diff
@@ -220,6 +220,13 @@ export const publisherDispatcher = () => {
     }
   );
 
+  // get bot status from target publisher
+  const getPublishStatusV2 = useRecoilCallback(
```

Comment:
If its a refactor why use V2? Can we just update the original functions for getPublishStatus?

Reply:
Because `BotRunTimeStatus` use the original function. The refactor move the http call to the polling, but `BotRunTimeStatus` still need the http call in the function. So I create a V2 function for refactor